### PR TITLE
replace Xceed.Wpf.Toolkit with DotNetProjects.Extended.Wpf.Toolkit

### PR DESCRIPTION
**Pull request by pinzart90**
License info:
Xceed introduced the non-commercial license on https://github.com/xceedsoftware/wpftoolkit/releases/tag/3.7.0 
DotNetProjects.Extended.Wpf.Toolkit : last merge from xceed was from release 3.6.0 (https://github.com/dotnetprojects/WpfExtendedToolkit/commit/66438b11f3b313f1d180b225a39c068001acc7ee)

### CODE REVIEW

File: src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs [hunk 1]
```diff
@@ -1114,7 +1114,7 @@ public bool NodeAutocompleteMachineLearningIsBeta
         {
             get
             {
-                return DynamoModel.FeatureFlags.CheckFeatureFlag("NodeAutocompleteMachineLearningIsBeta", false);
+                return DynamoModel.FeatureFlags?.CheckFeatureFlag("NodeAutocompleteMachineLearningIsBeta", false) ?? false;
```
Comment (pinzart90) [inline src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs:1117]: this causes an exception and makes it harder to debug

File: src/Libraries/CoreNodeModelsWpf/Charts/Controls/PieChartControl.xaml.cs [hunk 2]
```diff
@@ -5,7 +5,6 @@
 using LiveCharts.Wpf;
 using System.ComponentModel;
 using System.Windows;
-using System.Web.ModelBinding;
```
Comment (pinzart90) [inline src/Libraries/CoreNodeModelsWpf/Charts/Controls/PieChartControl.xaml.cs:8]: this was causing an issue on dotnet6 (it is not really needed)

File: src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs (hunk 1 above)
Comment (mjkkirschner) [reply src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs:1117]: @pinzart90 what do you think about forcing this member to be nullable? It seems like the API change is worth the wasted dev and user crash time.

**Review by mjkkirschner — APPROVED** [review]

**Comment by mjkkirschner** [comment]
@pinzart90 Do we need to make changes to the license.txt, about box and third party wiki with the link to the license in the new repo?

**Review by sm6srw — APPROVED** [review]
LGTM!

**Review by aparajit-pratap — APPROVED** [review]

File: src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs (hunk 1 above)
Comment (pinzart90) [reply src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs:1117]: This API is only used in binding to a UI element. The binding is handled by a BooleanConverter which handles nullables the same way the function is written now.

Comment (mjkkirschner) [reply src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs:1117]: Let's leave it out of this PR.

**Comment by mjkkirschner** [comment]
@pinzart90 
Screenshot 2023-07-11 at 11 48 09 AM

This doesn't look correct? Also strangely there is no change to the net6 sln?

**Pull request merged**